File: it/versions/structures/default/dart/bar/v2_1.dart
class Other {
  String name21;

  Other(
    this.name21
  );

  static Other decode(dynamic _data_dyn) {
    if (!(_data_dyn is Map<String, dynamic>)) {
      throw 'expected Map<String, dynamic>, but got: $_data_dyn';
    }

    Map<String, dynamic> _data = _data_dyn;

    var name21_dyn = _data["name21"];
    if (name21_dyn == null) {
      throw "expected value but was null";
    }
    if (!(name21_dyn is String)) {
      throw 'expected String, but was: $name21_dyn';
    }
    final String name21 = name21_dyn;

    return Other(name21);
  }

  Map<String, dynamic> encode() {
    Map<String, dynamic> _data = Map();

    _data["name21"] = this.name21;

    return _data;
  }
}

File: it/versions/structures/default/python3/bar/v2_1.py
class Other:
  def __init__(self, _name21):
    self._name21 = _name21

  @property
  def name21(self):
    return self._name21

  @staticmethod
  def decode(data):
    f_name21 = data["name21"]

    if not isinstance(f_name21, str):
      raise Exception("not a string")

    return Other(f_name21)

  def encode(self):
    data = dict()

    if self._name21 is None:
      raise Exception("name21: is a required field")

    data["name21"] = self._name21

    return data

  def __repr__(self):
    return "<Other name21:{!r}>".format(self._name21)
